app/frontend/src/features/profile/hooks/useUpdateUserProfile.ts

```diff
@@ -14,21 +15,27 @@ interface UpdateUserProfileVariables {
 export default function useUpdateUserProfile() {
   const queryClient = useQueryClient();
   const history = useHistory();
+  const { data: user } = useCurrentUser();
   const userId = useAuthContext().authState.userId;
   const { mutate: updateUserProfile, status, reset } = useMutation<
     Empty,
     Error,
     UpdateUserProfileVariables
   >(({ profileData }) => service.user.updateProfile(profileData), {
     onError: (error, { setMutationError }) => {
+      console.error({
+        error,
+      });
```

Comment:
remove

Reply:
Oops. eslint can prevent this. Maybe I look to add a warning for console statements to the rules?

Reply:
I think it's okay, there are legitimate uses. Maybe once we have proper error tracking